src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,62 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import { SortType } from '../../types/Filter';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+type Props = {
+  onSelectStatus: (filterType: SortType) => void,
+  changeQuery: (value: string) => void,
```

Comment:
one more useful resource – https://jaketrent.com/post/naming-event-handlers-react
```suggestion
  onChangeQuery: (value: string) => void,
```